modules/home/carousel/portfolios/DynamicPortfolioCarousel.tsx

```diff
@@ -122,12 +124,43 @@ const DynamicPortfolioCarousel = () => {
     ],
   };
 
+  if (error) {
+    return null;
+  }
+
   return (
     <>
-      {slides.length > 0 && (
-        <div className="overflow-hidden p-2 w-full mx-0 mt-[0]">
-          <Slider {...settings}>
-            {slides.map((logo, index) => (
+      <div className="overflow-hidden p-2 w-full mx-0 mt-[0]">
+        <Slider {...settings}>
+          {slides?.map(
+            (
+              logo: {
+                section: string;
+                src: any;
+                name:
```

Comment:
add the type to the dat object instead

the type union you are using is unnecessary

string|undefined is unnecessary

Reply:
fied